environment/commands/clean.js

```diff
@@ -17,13 +17,14 @@ class CleanCommand {
             });
         });
         try {
-            execSync('python3 ~/.rainbowup/nearup/nearup stop');
+            execSync('python3 ~/.rainbowup/nearup/main.py stop');
         } catch (err) {
-            console.log(`Error stopping nerup ${err}`);
+            console.log(`Error stopping nearup ${err}`);
         }
-        console.log('Cleaning ~/.rainbowup , ~/.nearup , and ~/.near directories...');
-        execSync('rm -rf ~/.rainbowup && rm -rf ~/.nearup && rm -rf ~/.near');
+        console.log('Cleaning ~/.rainbowup directory...');
+        execSync('rm -rf ~/.rainbowup');
         console.log('Cleaning done...');
+        process.exit(0)
```

Comment:
without this, index.js clean always hang on first clean. This is problematic on ci